### ISSUE TITLE
ibus popup menu does not *seem* to respect xinerama

### ISSUE THREAD

**Issue by fujiwarat** [ibus/ibus#1594]
```
OS (Linux distributions, UNIX or ...): Ubuntu 12.04.1 LTS
Architecture (i386, x86_64): x86_64
IBus version: 1.4.1
Input method name and version: IBus-Anthy 1.2.6
Python version: 2.7.3
dbus version: 1.4.18
dbus-python version: (unknown)
gtk version (if bug is about gtk applications): (Gnome-terminal, probably gtk 3)

What steps will reproduce the problem?
1. Set up dual monitor, with different resolution (1680x1050 and 1920x1080) with the
top aligned, IE so there's a gap on the bottom of the 1680x1050
2. Fullscreen gnome-terminal on the smaller monitor (1680x1050)
3. Run irssi, input should be on bottom row of gnome-terminal
4. enter ibus-anthy and type something, say "わたし" then press space to change it to
kanji then use the up/down arrows to bring up a menu for selecting which one to convert
to.
5. The menu is displaying below the input field, we can now only see a couple of pixels
on screen, the rest is off screen.

What is the expected output?

The expected output is for the menu to display *above* the word being converted, this
works properly on the (1920x1080) monitor, so it seems to indicate some form of xinerama
issue with ibus menu dropdown.
```

Original issue reported on code.google.com by `pharaun666` on 2013-01-15 23:00:55

**Comment by fujiwarat** [comment]
Are you still able to reproduce your issue with ibus 1.5.10?
We fixed issue 1676 but not sure about the xinerama issue.

**Comment by koterpillar** [comment]
I have the same, or perhaps similar, issue on IBus 1.5.11.

I have a multiple monitor configuration, arranged as shown: 
![screenshot from 2015-10-26 10-53-35](https://cloud.githubusercontent.com/assets/140276/10718847/d3c785d4-7bcf-11e5-8ee2-334123a01ef8.png)

Now, whenever I'm typing at the bottom of the large monitor (DP-0 in the picture), the candidate window goes off the screen, with only a few pixels visible.

If I set the monitors' bottom edge to align, the candidate window is properly positioned above the cursor.

**Comment by koterpillar** [comment]
Fixed this (at least on Gtk3) in #1824.
